src/sentry/api/serializers/base.py

```diff
@@ -35,8 +41,10 @@ def wrapped(cls):
 
 
 class Serializer(object):
-    def __call__(self, *args, **kwargs):
-        return self.serialize(*args, **kwargs)
+    def __call__(self, obj, attrs, user):
+        if not obj:
```

Comment:
because ne is not the same as None

i.e. an object can override that value and that doesnt mean it shouldnt serialize